oneview_redfish_toolkit/api/processor.py

```diff
@@ -45,11 +45,11 @@ def __init__(self, server_hardware, processor_id):
         self.redfish["@odata.type"] = self.get_odata_type()
         self.redfish["Id"] = processor_id
         self.redfish["Name"] = "Processor " + processor_id
+        state, health = \
+            status_mapping.get_redfish_status_struct(server_hardware["status"])
```

Comment:
Processor also map to server hardware, thus we can update State and Health as discussed and reviewed in the spreadsheet.